starter-code/iteration-1/main.ts

```diff
@@ -1,6 +1,47 @@
 // Iteration 1
 // 1. Create an array of strings
 
+// Iteration 1
+// 1. Create an array of strings
+let tasks: string[] = [];
+
+// 2. Create a function to add a tasks in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+
+function addTask(taskToBeAdded: string) {
+	tasks.push(taskToBeAdded);
+	console.log(`${taskToBeAdded} has been added to the list!`);
+	console.log(tasks.length);
+	return tasks.length;
+}
+
+// 3. Create a function to list all tasks, it must show in the console each tasks.
+
+function listAllTasks() {
+	tasks.forEach(singleTask => {
+		console.log(singleTask);
+	});
+};
+
+// 4. Create a function to delete a tasks, you must find the tasks inside the array and delete it.
+
+function deleteTask(taskToBeDeleted: string) {
```

Comment:
You need to add `:number` --> `function deleteTask(taskToBeDeleted: string): number {`